### PR TITLE
Update README to allow support for custom branding and enforce reference to original project page

### LINKED ISSUE #38: Update README to allow support for custom branding and enforce reference to original project page

### PR DESCRIPTION
**Pull request by the-c0d3br34k3r**
## Description
Closes #38.

Update README to allow support for custom branding (Logo & Title) along with requirement to link back to original project.

## Author's Checklist
- [x] **These changes don't break existing dependants.** If they do, and you suggest to proceed, 
please explain why.

## Reviewer's Checklist
- [ ] **All classes have documentation comments.** - NA
- [ ] **All methods have documentation comments.** - NA
- [ ] **No parameters are hardcoded.** Should be loaded as properties instead. If there needs to be hardcoded properties, explain why. - NA
- [ ] **All information to be logged/displayed to the user are internationalized.** If not, explain why. - NA
- [x] **README.md has been updated if needed.**
- [ ] **pom.xml version is set to the latest date** If not, explain why. - NA

## Assignee's Checklist
- [x] **All GitHub action checks have passed.**
- [x] **All reviewers have approved.**
- [x] **Relevant documentation has been updated if needed.**
- [ ] **pom.xml version is set to the latest date** If not, explain why. N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,20 +4,14 @@
...
- - Firefox
+- Google Chrome
+- Firefox
 
-### Limitations
- - Maximum of 10 files can be sanitized at a time.
- - Each file cannot exceed 50 MB.
- - Only HAR files are supported.
+At the moment, it sanitizes HAR files.<br>
+Multiple files can be sanitized at the same time.<br>
```
Comment (silvs110) [inline README.md:13]: Should we provide a the max number of supported files here?

**Review by silvs110 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (the-c0d3br34k3r) [reply README.md:12]: Yes, have added it.

File: README.md [hunk 2]
```diff
@@ -31,5 +25,45 @@ To run tests, run `run_tests`.
...
+<br>
+You can host a custom version of this project with your own set of rules by:
+#### Forking this repository
+You can fork this repository to maintain you own version of this project.<br>
+Be sure to rebase on the latest changes periodically to pull in new bugfixes and features.
+
+#### Adding/Updating the rules.
+- The rule files can be found in the `rules` directory.<br>
+- There are separate rule files for each file format (Eg: `har.yaml`)<br>
+- For more info on writing rules refer [rules/README.md](rules/README.md).<br>
```
[the remark this answers is not in the document]
Comment (the-c0d3br34k3r) [reply README.md:51]: Good catch. Have added it.

File: rules/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+```
+description: <Description of the file format and some info on the find of info sanitized.>
+format: <json>
+rules:
+    <json_path_pattern>:
+        description: <Information on what this rule sanitizes>
+        action: <contextual_replacement|remove>
+    ...
+```
+For an actual rule file, refer [har.yaml](har.yaml)
```
Comment (silvs110) [inline rules/README.md:21]:
```suggestion
For an actual rule file, refer to [har.yaml](har.yaml)
```

File: rules/README.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+# Sanitization rules
+
+The rules contain the patterns/specific fields and the approach to sanitize them.
+
+## Rule file
+Each rule file is the yaml format stored as `rules/<file_extension>.yaml`.<br>
```
Comment (silvs110) [inline rules/README.md:6]:
```suggestion
Each rule file is in the yaml format stored as `rules/<file_extension>.yaml`.<br>
```

File: rules/README.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+For example, the rule file for HARs will be `rules/har.yaml`
+Each file can have one or more rules.
+
+### File format
+The format is as follows:
+```
+description: <Description of the file format and some info on the find of info sanitized.>
+format: <json>
+rules:
+    <json_path_pattern>:
```
Comment (silvs110) [inline rules/README.md:16]: Will it always be a json_path_pattern

**Review by silvs110 — CHANGES_REQUESTED** [review]

File: rules/README.md (hunk 5 above)
Comment (the-c0d3br34k3r) [reply rules/README.md:16]: Atm, yes

**Review by silvs110 — APPROVED** [review]

**Pull request merged**